src/auth.c

```diff
@@ -159,6 +178,10 @@ unsigned IsClientAuthorized(int socket, const char* action, const char* reader)
 	return ret;
 }
 
+#elif defined(HAVE_POLKIT)
+
+#error polkit is enabled, but no socket cred implementation for this platform
+
 #else
 
 unsigned IsClientAuthorized(int socket, const char* action, const char* reader)
```

Comment:
Given the above, `#error polkit is enabled, but no socket cred implementation for this platform`, I'm not sure if this will matter or not. But I wonder if this function still has an effect, if we can log some kind of message to stderr stating that the allow-all `IsClientAuthorized()` function is being used.

May be outside the scope of this PR, but wanted to suggest it as a way to make it more obvious if others run into this.

Reply:
The allow-all one is what you should get when you build with -Dpolkit=false to avoid sprinkling more HAVE_POLKIT around and allow other authorization mechanisms to be plugged in inside.  Its use is actually expected rather than an error in the default configuration, and that seems more or less fine.

Reply:
Ah, gotcha. Thank you for explaining!